types.js

```diff
@@ -11,6 +11,9 @@ export type TONConfigData = {
     queriesWsServer?: string,
     log_verbose?: boolean,
     tracer?: ?Object, // opentracing.Tracer
+    authorization?: string,
```

Comment:
authorization??
it is superseded by `accessKey`
and remove please obsolete `*server` parameters

Reply:
Done